include/seqan3/alignment/pairwise/alignment_range.hpp

```diff
@@ -73,19 +65,18 @@ class alignment_range
     ~alignment_range() = default; //!< Defaulted.
 
     //!\brief Explicit deletion to forbid copy construction of the underlying executor.
-    explicit alignment_range(alignment_executor_type const & _alignment_executor) = delete;
+    explicit alignment_range(alignment_executor_type const & alignment_executor) = delete;
 
     /*!\brief Constructs a new alignment range by taking ownership over the passed alignment buffer.
-     * \tparam _alignment_executor_type The buffer type. Must be the same type as `alignment_executor_type` when
-     *                                  references and cv-qualifiers are removed.
```

Comment:
why remove the type description? I think it does not copy from the class template type description automatically does it?

Reply:
Because the constructor is not a template function. It get's an explicit type, which is documented in the class documentation